maven-plugins/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/AbstractBndMavenPlugin.java

```diff
@@ -258,6 +258,15 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 			// handle projects with no output directory, like
 			// 'test-wrapper-bundle'
 			if (classesDir.isDirectory()) {
+
+				// delete existing MANIFEST.MF from previous build
+				// target/classes/META-INF is output and should not be picked up
+				File existingMetaInf = new File(classesDir, "META-INF/MANIFEST.MF");
+				if (existingMetaInf.exists()) {
+					logger.info("Delete existing manifest from previous build: " + existingMetaInf);
+					IO.delete(existingMetaInf);
```

Comment:
This does break a potential use case where the user caused a manifest file to be written to target/classes which they want to be input to the bnd process. For example, they could unzip an existing jar into target/classes and run bnd to make it a bundle. This change would prevent that use case. I think you probably need to make some configuration option which can be set to always delete any extant manifest in target/classes.

Reply:
Thanks. I will add a config option.

Reply:
I added a new option `deleteExistingManifest` (default `false`)


@Marcono1234 with this configuration repeated 
mvn process-classes
mvn process-classes

```
<plugin>
                <groupId>biz.aQute.bnd</groupId>
                <artifactId>bnd-maven-plugin</artifactId>
                <version>7.1.0-SNAPSHOT</version>
                <executions>
                    <execution>
                        <goals>
                <goal>bnd-process</goal>
            </goals>
                        <configuration>
                        	<deleteExistingManifest>true</deleteExistingManifest>
                            <bnd><![CDATA[
                                -exportcontents: example.a, example.b, example.c
                                Import-Package: example.c;version="${range;[==,=+)}", *
                            ]]></bnd>
                        </configuration>
                    </execution>
                </executions>
            </plugin>
```